<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObservationsTab</class>
 <widget class="QWidget" name="ObservationTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>386</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,1,0,1,0,1">
     <item row="0" column="0">
      <widget class="QLabel" name="sessionNameLabel">
       <property name="text">
        <string>Session Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="sessionNameComboBox"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="cameraLabel">
       <property name="text">
        <string>Camera:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="cameraComboBox"/>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="telescopeLabel">
       <property name="text">
        <string>Telescope:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QComboBox" name="telescopeComboBox"/>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="filterLabel">
       <property name="text">
        <string>Filter:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QComboBox" name="filterComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="objectLabel">
       <property name="text">
        <string>Object:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QComboBox" name="objectComboBox"/>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="imageCountLabel">
       <property name="text">
        <string>Image Count:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QSpinBox" name="imageCountSpinBox">
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="exposureLengthLabel">
       <property name="text">
        <string>Exposure Length (s):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QSpinBox" name="exposureLengthSpinBox">
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="commentsLabel">
       <property name="text">
        <string>Comments:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="5">
      <widget class="QLineEdit" name="commentsLineEdit">
       <property name="placeholderText">
        <string>Enter comments (optional)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6" colspan="2">
      <widget class="QPushButton" name="addObservationButton">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Add Observation</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="filterAndTableLayout" stretch="0,1">
     <item>
      <widget class="QListView" name="observationFilterListView">
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="observationsTable">
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>70</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Session Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Object</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Camera</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Telescope</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Filter</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Images</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Exposure (s)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Total
Exposure (s)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Moon
Illumination</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Angular
Separation</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Comments</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="editObservationButton">
       <property name="text">
        <string>Edit Selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="exportObservationButton">
       <property name="styleSheet">
        <string notr="true">padding-right: 20px;</string>
       </property>
       <property name="text">
        <string>Export...</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteObservationButton">
       <property name="text">
        <string>Delete Selected</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
